site/content/docs/CICD/vercel.md

```diff
@@ -12,54 +12,50 @@ aliases:
   - "/docs/integrations/vercel/"
 ---
 
-If you are using Vercel to develop, preview, and ship your application, you can natively integrate with Checkly by [installing the official integration](https://vercel.com/integrations/checkly) from the Vercel Marketplace.
+If you are using Vercel to develop, preview, and ship your application, you can natively integrate with Checkly by installing [the official integration](https://vercel.com/integrations/checkly) from the Vercel Marketplace. 
+
+Checkly offers two integration paths:
+- Native integration, **if you don't yet have a Checkly account**. With the native integration, user access and billing is managed through Vercel. If you are new to Vercel's native integrations you can read more about them on the [Vercel documentation page](https://vercel.com/docs/integrations#native-integrations).
```

Comment:
@drakirnosslin I think this is a bit too oversimplified. You can also still create a new account with the connected integration.